<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RampQueuer</class>
 <widget class="QWidget" name="RampQueuer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ramp Queuer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2" rowspan="9">
      <widget class="QListWidget" name="listToQueue"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkPrependRamp">
         <property name="text">
          <string>Prepend Ramp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushChangeFile">
         <property name="text">
          <string>Change Prepend Ramp</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushUpdateServerQueue">
       <property name="text">
        <string>Update Server Queue</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushChangeMainRamp">
       <property name="text">
        <string>Change Main Ramp</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushPauseAfterCurrent">
       <property name="text">
        <string>Pause After Current</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditMainRamp"/>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushClearServerQueue">
       <property name="text">
        <string>Clear Server Queue</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Server IP:Port</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="serverIPAndPort"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>219</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Local</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushAbortCurrentRun">
       <property name="text">
        <string>Abort Current Run</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Local ramps queue:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushQueueToServer">
       <property name="text">
        <string>Transfer queue to server</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ramps Queued on Server</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Server Logs:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushAddCurrent">
         <property name="text">
          <string>Add Current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Reps:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinNumberReps">
         <property name="toolTip">
          <string>Number of reps</string>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="push1DScan">
         <property name="text">
          <string>Add 1D Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push2DScan">
         <property name="text">
          <string>Add 2D Scan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPrependRamp">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="9">
      <widget class="QListWidget" name="listQueued"/>
     </item>
     <item row="1" column="4" rowspan="9">
      <widget class="QTextEdit" name="textServerMesg">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushClearLocalQueue">
       <property name="text">
        <string>Clear Local Queue</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushChangeFile</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleChangePrependRamp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAddCurrent</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleAddCurrent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push1DScan</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handle1DScanPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushQueueToServer</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleQueueToServerPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushUpdateServerQueue</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleUpdateServerQueuePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushChangeMainRamp</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleChangeMainRamp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushPauseAfterCurrent</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handlePauseAfterCurrent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClearServerQueue</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleClearServerQueue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAbortCurrentRun</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleAbortCurrentRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushStart</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handlePushStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClearLocalQueue</sender>
   <signal>clicked()</signal>
   <receiver>RampQueuer</receiver>
   <slot>handleClearLocalQueue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleChangePrependRamp()</slot>
  <slot>handleAddCurrent()</slot>
  <slot>handle1DScanPressed()</slot>
  <slot>handleQueueToServerPressed()</slot>
  <slot>handleUpdateServerQueuePressed()</slot>
  <slot>handleChangeMainRamp()</slot>
  <slot>handlePauseAfterCurrent()</slot>
  <slot>handleClearServerQueue()</slot>
  <slot>handleAbortCurrentRun()</slot>
  <slot>handlePushStart()</slot>
  <slot>handleClearLocalQueue()</slot>
 </slots>
</ui>
